RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,38 +8,69 @@
 
 import Cocoa
 
+public protocol EntitiesViewDataSource : NSObjectProtocol {
+    func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    func entitiesView(tableView:NSTableView, viewForTableColmn tableColumn: NSTableColumn?, row: Int) -> NSView?
+}
+
+public protocol EntitiesViewDelegate : NSObjectProtocol {
+    func addEntityInEntitiesView(entitiesView:EntitiesView)
+    func removeEntityInEntitiesView(entitiesView:EntitiesView)
+    func entitiesViewSelectionDidChange(notification: NSNotification)
+}
+
 @IBDesignable
-class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource {
+public class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource {
     
     @IBOutlet var tableView:NSTableView!
-    
     @IBOutlet var addButton:NSButton!
     @IBOutlet var removeButton:NSButton!
     
-    override func nibDidLoad() {
+    var dataSource:EntitiesViewDataSource?
+    var delegate:EntitiesViewDelegate?
+    
+    override public func nibDidLoad() {
```

Comment:
if we aren't using the override method for anything lets not keep it in the file